### PR TITLE
improvements from internal review

### PR DESCRIPTION
**Pull request by Rick-Kirkham**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[bf426fe](https://github.com/OfficeDev/office-js-docs-pr/commits/bf426fe750ec03cf9b8b43db83620b551668e64f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/auth-external-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/auth/docs/develop/auth-external-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/overview-authn-authz.md](https://github.com/OfficeDev/office-js-docs-pr/blob/auth/docs/develop/overview-authn-authz.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C6%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201909062214333814-1294%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=X4aH568BV3m2Aop9uER35DtDp8G9AYr4zp0FzamXCI8%3D&st=2019-09-06T22%3A10%3A43Z&se=2019-10-07T22%3A15%3A43Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[8eafeff](https://github.com/OfficeDev/office-js-docs-pr/commits/8eafeff3045de7676974f0205a011dc7c52aecc2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/auth-external-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/auth/docs/develop/auth-external-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/overview-authn-authz.md](https://github.com/OfficeDev/office-js-docs-pr/blob/auth/docs/develop/overview-authn-authz.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C6%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201909062223151096-1294%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=WulW4GyM3hBNZ2xatx9mONGM2H51Fg69tsJ7gE5WpR4%3D&st=2019-09-06T22%3A19%3A20Z&se=2019-10-07T22%3A24%3A20Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/develop/overview-authn-authz.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ localization_priority: Priority
 
 # Overview of authentication and authorization in Office Add-ins
 
-Web applications and, hence, Office Add-ins allow anonymous access by default, but you can require users to authenticate with a login. In particular, you can require that your users be logged in with either a Microsoft Account or a Work or School (Office 365) account. This task is called user authentication because it enables the add-in to know who the user is.
+Web applications and, hence, Office Add-ins allow anonymous access by default, but you can require users to authenticate with a login. For example, you can require that your users be logged in with either a Microsoft Account or a Work or School (Office 365) account, or other common accounts. This task is called user authentication because it enables the add-in to know who the user is.
```
Comment (AlexJerabek) [inline docs/develop/overview-authn-authz.md]: This sentence is a little awkward with the "either... or … or". Consider:
"For example, you can require that your users be logged in with a Microsoft Account, an Office 365 Work or School account, or another common account."

**Comment by o365devx** [comment]
Docs Build status updates of commit _[50c0d92](https://github.com/OfficeDev/office-js-docs-pr/commits/50c0d92e8889302e59ee5e48feebf4b8a3c9c3f5)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/auth-external-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/auth/docs/develop/auth-external-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/overview-authn-authz.md](https://github.com/OfficeDev/office-js-docs-pr/blob/auth/docs/develop/overview-authn-authz.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C9%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201909091724533374-1294%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=FqtmqHlAXutSYZCb5DHXqM2xVAzUThmUhJ1fE4qZGx0%3D&st=2019-09-09T17%3A20%3A59Z&se=2019-10-10T17%3A25%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**